src/Processors/Formats/Impl/ArrowColumnToCHColumn.cpp

```diff
@@ -584,7 +588,6 @@ void ArrowColumnToCHColumn::arrowColumnsToCHChunk(Chunk & res, NameToColumnPtr &
         }
 
         column.type = header_column.type;
-        num_rows = column.column->size();
```

Comment:
Why was it deleted? Now num_rows is always 0 and it will break code in line 594 `res.setColumns(columns_list, num_rows);`

Reply:
I think we can get `num_rows` from arrow columns before the loop

Reply:
Sorry,  it is a mistake, it's fixed now.